lookbook/docs/patterns/14-danger-dialog.md.erb

```diff
@@ -16,10 +21,10 @@ The FeedbackDialog is a variation of FeedbackDialog. It consists of:
 - A heading with a confirmation question such as "Permanently delete?"
 - A message explaining the consequences
```

Comment:
```suggestion
- A title: a short description of the dialog action. This should be written in a neutral tone. The title is currently only used for accessibility purposes. Examples:
    - "Delete work package"
    - "Delete project"
- A heading: a question inviting the user to take a decision, This should be written in an active tone. You may also to give the full descriptive name of the resource on which the action will take place (e.g. "meeting occurrence" rather than just "meeting"). Examples:
    - "Permanently delete this work package?"
    - "Cancel this meeting occurrence?"
- A message explaining the consequences
```